qdisc_linux.go

```diff
@@ -375,6 +382,16 @@ func (h *Handle) QdiscList(link Link) ([]Qdisc, error) {
 					if err := parseTbfData(qdisc, data); err != nil {
 						return nil, err
 					}
+				case "hfsc":
+					log.Println(attr.Value)
+					data, err := nl.ParseRouteAttr(attr.Value)
+					log.Println(data)
```

Comment:
I believe this was logging during testing. Please remove

Reply:
You are correct, should be removed now